applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -471,7 +471,7 @@ static void config_get(cJSON *parent, struct cloud_data_cfg *data)
 	cJSON *active_wait = cJSON_GetObjectItem(parent, CONFIG_ACTIVE_TIMEOUT);
 	cJSON *move_res = cJSON_GetObjectItem(parent, CONFIG_MOVE_RES);
 	cJSON *move_timeout = cJSON_GetObjectItem(parent, CONFIG_MOVE_TIMEOUT);
-	cJSON *acc_thres = cJSON_GetObjectItem(parent, CONFIG_ACC_THRESHOLD);
+	cJSON *acc_thres = cJSON_GetObjectItem(parent, CONFIG_ACC_ACT_THRESHOLD);
```

Comment:
Missing inactivity threshold and inactivity timeout.

Reply:
should be added now